templates/base.html

```diff
@@ -1,4 +1,4 @@
-<!DOCTYPE html>
+<!DOCTYPE html
```

Comment:
?

Reply:
Whoops, should have caught that..

https://github.com/the-blue-alliance/the-blue-alliance/commit/7f0783285798349fd29ff42853b61ee4ed2fddc5